### PR TITLE
✨ Add Converter Component

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
- 👽 update data attributes 
- ✨ add input components
- ✨ add Calculator page
- ✨ fetch data from API
- ✨ populate converter options with name attributes keys

### CODE REVIEW

File: components/calculator/rows/row.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import styles from "./row.module.css";
```
Comment (phlgr) [inline components/calculator/rows/row.tsx:1]: This import should be removed if styles isn't used in your component or use the styles to style your component

File: pages/Calculator.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import styles from "../styles/Calculator.module.css";
```
Comment (phlgr) [inline pages/Calculator.tsx:1]: Same here

**Review by phlgr — COMMENTED** [review]
GJ 🚀

**Review by mewiemers — COMMENTED** [review]
LGTM 🍰

File: components/calculator/rows/row.tsx (hunk 1 above)
Comment (SilviaBarcellini) [reply components/calculator/rows/row.tsx:1]: Will do, thanks

**Pull request merged**